templates/web/base/report/new/category.html

```diff
@@ -5,7 +5,7 @@
     <label for='form_category' id="form_category_label">
         [%~ loc('Category') ~%]
     </label>[% =%]
-    <select class="form-control" name='category' id='form_category'>
+    <select class="form-control" name='category' id='form_category' data-role='[% c.user_exists AND c.user.has_body_permission_to('planned_reports') ? 'inspector' : 'user'  %]' data-body='[% c.user.from_body.name %]'>
```

Comment:
I don't think the first `c.user_exists` call is necessary either looking at it, as even when logged out, it still returns `true`. Not sure if that's the case with all the cobrands though